factorio-map-process.sh

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env bash
+
+## Defaults
+SERVER_BASE_PATH=/srv/factorio/maps
+MAP_TILES_PATH=images
+FACTORIO_MAP_SCRIPT="$(dirname "${BASH_SOURCE[0]}")/factoriomap.py"
+TARFILE=
+REDUCE=0
+
+function usage() {
+    cat <<-EOF
```

Comment:
I learned something today, so thanks.
The `-` in the `<<-EOF` string means something:

https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Here-Documents
> If the redirection operator is ‘<<-’, then all leading tab characters are stripped from input lines and the line containing delimiter. This allows here-documents within shell scripts to be indented in a natural fashion.

This doesn't seem to apply here, since you aren't using tab characters at all (it ignores space chars), and you haven't indented in this string. I guess remove the `-`.
```suggestion
    cat <<EOF
```